sunshine/confighttp.cpp

```diff
@@ -250,7 +250,8 @@ void saveApp(resp_https_t response, req_https_t request) {
       fileTree.erase("apps");
       fileTree.push_back(std::make_pair("apps", newApps));
     }
-    pt::write_json(config::stream.file_apps, fileTree);
+    std::string sunshine_apps_path = SUNSHINE_CONFIG_DIR "/" APPS_JSON;
```

Comment:
`config::stream.file_apps` is configurable by the user.
Therefore, it shouldn't be changed at runtime